Implement support for sticky file headers

ericcornelissen opened the pull request:
Closes #455 

Adds support for sticky file headers through an optional CSS class and an option on `Diff2HtmlUI`. By default, sticky file headers are not enabled (I'm happy to change this based on feedback). I documented this new option in the README. I did not add tests as I did not spot a test suite for `Diff2HtmlUI`, please let me know if (and where) I should add some tests :slightly_smiling_face: 

Further details can be found in 7d4a5dce6f5e17c6ae2d09f62d503178bfe81d3f

rtfpessoa commented on README.md:
```diff
@@ -165,6 +166,7 @@ highlightCode(): void
 - `fileListToggle`: allow the file summary list to be toggled: `true` or `false`, default is `true`
 - `fileListStartVisible`: choose if the file summary list starts visible: `true` or `false`, default is `false`
 - `fileContentToggle`: allow each file contents to be toggled: `true` or `false`, default is `true`
+- `stickyFileHeader`: make file headers sticky: `true` or `false`, default is `false`
```
```suggestion
- `stickyFileHeaders`: make file headers sticky: `true` or `false`, default is `false`
```

rtfpessoa commented on README.md:
```diff
@@ -156,6 +156,7 @@ draw(): void
 synchronisedScroll(): void
 fileListToggle(startVisible: boolean): void
 highlightCode(): void
+stickyFileHeader(): void
```
```suggestion
stickyFileHeaders(): void
```

rtfpessoa reviewed (commented):
👋 

> By default, sticky file headers are not enabled (I'm happy to change this based on feedback).

Looks great to me. Thanks a lot for the contribution.
From my side I would enable it by default in Diff2HtmlUI.

> I documented this new option in the README. I did not add tests as I did not spot a test suite for Diff2HtmlUI, please let me know if (and where) I should add some tests 🙂

No tests for the JS part unfortunately. I did some manual ones and looks good.

Just left some comments about the docs.

ericcornelissen commented:
> Looks great to me. Thanks a lot for the contribution.
> From my side I would enable it by default in Diff2HtmlUI.

Sounds good to me, updated :slightly_smiling_face:

> No tests for the JS part unfortunately. I did some manual ones and looks good.

:+1: 

> Just left some comments about the docs.

Thanks for spotting those mistakes, updated as well.
